services/discordService.js

```diff
@@ -85,10 +85,27 @@ const setUserDiscordNickname = async (userName, discordId) => {
   }
 };
 
+const markUserVerified = async (discordId) => {
+  try {
+    const unverifiedRoleId = config.get("discordUnverifiedRoleId");
+
+    if (!discordId) {
+      throw Error;
+    }
```

Comment:
Do we need to have this if check here?
I think, if the function accepts the discordId as parameter, and if we omit that parameter, we wont be able to call the function right?
And i think this if check is checking for whether discordId is there or not. 

I think we will never actually hit this condition the way this function is implemented, hence we do not need this here. Please mention if i have missed something.